.circleci/config.yml

```diff
@@ -1,12 +1,12 @@
-version: 2
+version: 2.1
```

Comment:
Is there a reason for 2.1? Is that the version that includes the feature I believe they call "build processing", which is essentially parameterized jobs? Are we using any of those features?

**Edit** Oh, I see we're using `orbs` now 😮

Reply:
Yes, exactly.

If there is a reason not to use the newer version I could refactor the config, but this makes it simpler.